src/main/groovy/com/netgrif/application/engine/petrinet/domain/dataset/logic/action/ActionDelegate.groovy

```diff
@@ -458,59 +474,104 @@ class ActionDelegate {
         }]
     }
 
-    protected SetDataEventOutcome createSetDataEventOutcome() {
-        return new SetDataEventOutcome(this.useCase, this.task.orElse(null))
+    /**
+     * Changes behavior of given fields (given by field IDs) on given tasks (given by task IDs)  containing given fields if certain condition is being met.
+     * <br>
+     * Example 1:
+     * <pre>
+     *     taskRef: f.taskRef_0;
+     *     def taskIds = [taskRef.value[0]] as List
+     *     makeInterProcess ["field_Id"], editable on taskIds when {true}
+     *
+     * </pre>
+     * This code will change the behavior of fields <i>text</i> and <i>anotherText</i> to <i>visible</i> on given transition when field's <i>condition</i> value is equal to <i>true</i>.
+     *
+     * @param list of field IDs which behaviour will be changed
+     * @param behavior one of initial, visible, editable, required, optional, hidden, forbidden
+     */
+    def makeInTask(List<String> fieldIds, Closure behavior) {
```

Comment:
1. Why is this a separate function and not part of original `make`?
2. Name of the function in javadoc is `makeInterProcess` not  `makeInTask`
3. Message of thrown exception is also misleading with `make` as the called method

Reply:
`make` has the same argument types than this function, so they cannot have common name. Other things will be correected.